docs/content/configuration.md

```diff
@@ -704,7 +704,8 @@ included in the actual bundle gzipped tarball.
 | Field | Type | Required | Description |
 | --- | --- | --- | --- |
 | `decision_logs.service` | `string` | No | Name of the service to use to contact remote server. If no `plugin` is specified, and `console` logging is disabled, this will default to the first `service` name defined in the Services configuration. |
-| `decision_logs.partition_name` | `string` | No | Path segment to include in status updates. |
+| `decision_logs.partition_name` | `string` | No | Path segment to include in status updates. This field will take precedence over `decision_logs.resource` if set. |
```

Comment:
I suggest we just deprecate `decision_logs.partition_name` and recommend `decision_logs.resource`. Concretely, just add a note in the description mentioning it's deprecated.

Reply:
done!